### PR TITLE
Minor footer appearance tweaks

### PR DESCRIPTION
**Pull request by janfaracik**
Minor footer tweaks to bring it more inline with the ongoing design work.

* The links now uses the .jenkins-button classes, giving them better focus states and larger hitboxes
* The version number is clickable, offering helpful links to About Jenkins, Get involved, and the Jenkins website
* The footer hides on mobile now

image

image

### Testing done

* Footer works as before
* Version dropdown works

### Proposed changelog entries

* Minor footer appearance tweaks

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] The Jira issue, if it exists, is well-described.
- [ ] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)).
  - Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [ ] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.

### Desired reviewers

@jenkinsci/sig-ux 

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7989"></a>

### CODE REVIEW

**Comment by timja** [comment]
cc @imonteroperez @Vlatombe in case this impacts your product.

**Comment by NotMyFault** [comment]
Screenshot 2023-05-16 at 22 26 10

The simple-theme plugin might need an update too.

**Comment by timja** [comment]
> Screenshot 2023-05-16 at 22 26 10
> 
> The simple-theme plugin might need an update too.

The simple theme plugin doesn't do that as far as I know. it will be a third party theme that is not maintained from what I can see. I wouldn't worry about it.

File: core/src/main/resources/lib/layout/layout.jelly
Comment (timja) [reply core/src/main/resources/lib/layout/layout.jelly:241]: actually manage or system read

Comment (NotMyFault) [reply core/src/main/resources/lib/layout/layout.jelly:241]: That's what ci.j reported 👀

Comment (janfaracik) [reply core/src/main/resources/lib/layout/layout.jelly:241]: Updated, let me know if that's the right permission 👍

**Review by NotMyFault — APPROVED** [review]

**Review by timja — APPROVED** [review]
/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Pull request merged**
